web/styles/drafts.css

```diff
@@ -60,6 +60,7 @@
             font-size: 1.1em;
             line-height: normal;
             margin-bottom: 5px;
+            white-space: pre-wrap;
```

Comment:
This should be `pre` to avoid the wrap and replace the overflow with ellipsis. 

<img width="359" alt="Screenshot 2024-06-23 at 3 17 17 PM" src="https://github.com/zulip/zulip/assets/25124304/a154eb2c-5c2a-4142-b374-17b49000312b">

Reply:
Fixed it!

Reply:
That is because the issue was coming from message header which I modified. This code points to the draft header above it.